README.md

```diff
@@ -2,30 +2,37 @@
 [WIP] A tool for C++ code modification to augment data for clone detection tools
 
 -------------
-## Usage
-1. Clone or download repo from GitHub
+## Installation
+Clone or download repo from GitHub
 ```(bash)
 git clone https://github.com/JetBrains-Research/gorshochek.git
 cd gorshochek
 ```
-2. Install clang with the desired version to use only inside `gorshochek`
+### Docker
+It is more preferable to run `gorshochek` using `Docker`. All necessary scripts to work with `Docker`
+are stored in `docker` folder. 
+
+To build docker image run:
+```(bash)
+./docker/build.sh
+```
+
+Then to run a container from just created image run:
+```(bash)
+./docker/build.sh
+```
+### Instlling on local machine
```

Comment:
I think we don't need instructions to local build since we have docker with the desired environment.
In the end, we understand what happened in docker and can guarantee workness. But if the users build all by themselves then we probably have to help them from time to time :)